doc/architecture/decisions/0002-godaddy-structured-json-logging-java.md

```diff
@@ -0,0 +1,31 @@
+# 2. Godaddy structured JSON logger for Java services
+
+Date: 17/08/2018
+
+## Status
+
+Proposed
```

Comment:
Do we need a status. Assuming if it's a PR it's a proposal and given it's merged it's accepted?